app/src/main/java/ch/epfl/sweng/zuluzulu/Fragments/ChatFragment.java

```diff
@@ -75,6 +79,7 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
 
         String collectionPath = CHANNEL_DOCUMENT_NAME + channel.getId() + "/" + MESSAGES_COLLECTION_NAME;
         collectionReference = db.collection(collectionPath);
+        mockableCollection = DatabaseFactory.getDependency().collection(collectionPath);
```

Comment:
Does it works ? :smiley:

Reply:
@Dahnepfl 
Yeah it seems to work, good job